ligo/stablecoin/spec.ligo

```diff
@@ -0,0 +1,227 @@
+// SPDX-FileCopyrightText: 2020 tqtezos
+// SPDX-License-Identifier: MIT
+
+/*
+ * Stablecoin smart-contract
+ */
+
+/* ------------------------------------------------------------- */
+
+/*
+ * Parameter
+ */
+
+type tokenId is nat
+
+
+
+type transferDescriptor is record
+  id          : tokenId
+; fromAddress : address
+; toAddress   : address
+; amount      : nat
+end
+
+type transferParams is record
+  batch    : list (transferDescriptor)
+; operator : address
```

Comment:
I didn't understand why the `operator` field is required here. It seems that it is not mentioned in the specification doc or the FA2 doc in the sections for `transfer` operation.

Reply:
Good catch